doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -220,16 +225,36 @@ Robot Framework 4.0 introduced new spec version `3`.
 
 .. note:: Robot Framework 4.0 introduced an updated xml format. It is
           incompatible with former versions. LibDoc can only read and write
-          new XML format.
+          new XML format version 3.
 
-          The `xml:html` format introduced in Robot Framework 3.2. has been
-          replaced by the option `-s --specdocformat <raw|html>`.
+          The `XML:HTML` format introduced in Robot Framework 3.2. has been
+          replaced by the format `LIBSPEC` ot the option combination
+          `--format XML --specdocformat HTML`.
 
           Including source information and spec version are new in Robot
           Framework 3.2 as well.
 
 __ https://en.wikipedia.org/wiki/XML_Schema_(W3C)
 
+Libdoc JSON spec files
+''''''''''''''''''''''
+
+Since Robot Framework 4.0 Libdoc can also generate documentation in JSON
+format that is suitable for external tools such as editors or web pages.
+It contains all the same information as the HTML format but in a machine
+readable format.
+
+Similar to XML spec files the JSON spec files contain all information and
+can also be used as input to Libdoc. From that format any other output format
+can be created. By default the library documentation strings are converted
+to HTML format within the json output file.
+
+The exact JSON spec file format is documented with an `JSON schema`__ (json)
```

Comment:
What that `(json)` at the end?

Reply:
🤷🏻‍♂️

It was there because (xsd) was there on xml schema... but you are right. makes no sense